### ISSUE TITLE
Bug: Command-ids should not contain spaces

### ISSUE THREAD

**Issue by chrisgrieser** [tadashi-aikawa/obsidian-another-quick-switcher#97]
One of the recent updates introduces the creation of custom quick switchers (which was awesome, btw!).

However, with the new commands, they now by default contain spaces (e.g. "Recent Search"). These are carried over to the command id: `obsidian-another-quick-switcher:search-command_Recent search`

having spaces in the command id is a problem, since it prevents assigning the command in the vimrc plugin via exmap:

```vim
" go = open quick Switcher
exmap quickSwitcher obcommand obsidian-another-quick-switcher:search-command_Recent search
nmap go :quickSwitcher
vmap go :quickSwitcher
```

Replacing all spaces in the command-id with hyphens would fix this issue.

**Comment by tadashi-aikawa** [comment]
Thank you for your report, @chrisgrieser :)

I agree with you. But I think it affects only vimrc plugin (because Obsidian accepts the key name that contains whitespace). And it affects users who use AQS.

So, I'll implement it in the next major release (maybe soon), which includes the features optimized for Obsidian v0.16.0 (ex: tabs). Probably, in v7.0.0-beta1 👍

**Comment by tadashi-aikawa** [comment]
Released in [7.0.0-beta1](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.0-beta1) 🚀

**Comment by tadashi-aikawa** [comment]
Released in [v7.0.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.0) 🚀

**Comment by chrisgrieser** [comment]
minor follow-up issue:
it seems that only the first space is replaced, which in some rare cases like the file name search is noticable:

```
obsidian-another-quick-switcher:search-command_recent-search
obsidian-another-quick-switcher:search-command_file-name search <-- still a space there
obsidian-another-quick-switcher:search-command_landmark-search
obsidian-another-quick-switcher:search-command_star-search
obsidian-another-quick-switcher:search-command_backlink-search

```

**Comment by tadashi-aikawa** [comment]
Thanks. Fix in [v7.0.1](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.1) 👍
